Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -154,12 +225,18 @@ public class KotlinTokenizer: SwiftTokenizer {
                         .joined(tokens: joinTokens))
                     .prefix(with: linebreak)
             }
+            var inheritanceTokens = [Token]()
+            if !inheritanceRange.isEmpty {
+                inheritanceTokens = Array(tokens[inheritanceRange])
+                tokens.removeSubrange(inheritanceRange)
+                bodyStart -= inheritanceTokens.count
+            }
+            inheritanceTokens = [] // clearing this for now, inheritance in data class doesn't seem to work?
```

Comment:
Can you please provide more info about this change? why remove the inheritance? it could be a protocol conformance as well